_api-files/embedding-api/objects/destinations/destination-object.md

```diff
@@ -0,0 +1,34 @@
+---
+content-type: "embed-object"
+endpoint: "destinations"
+
+title: "Destination"
+description: "Destinations are the data warehouses into which Stitch writes data."
+endpoint-url: "/destinations"
+version: "3"
+
+object-attributes:
+  - name: "id"
+    type: "integer"
+    description: "A unique identifier for this destination."
+
+  - name: "connection"
+    type: "destination-properties" # this = destination form properties?
```

Comment:
Can this be linked to the destination properties object section? If not, I'd suggest leaving the `type` property blank for this attribute and describing it in the `description`. Seeing "Destination-Properties" where I was expecting to see a datatype is a little unintuitive without additional context.

Reply:
I think this applies across the board for object properties.

Reply:
Yep - this is one of the things I'm still working on :)